app/styles/app.css

```diff
@@ -3,6 +3,7 @@
 @import "image-upload.css";
 @import "tasks.css";
 @import "sidebar.css";
+@import "get-started.css";
```

Comment:
We might want to use single quotes here 
https://cli.emberjs.com/release/advanced-use/stylesheets/

Reply:
075a915